### DISCUSSION TITLE
What do we miss for PHP8.1?

### DISCUSSION THREAD

**Discussion by fballiano** [OpenMage/magento-lts#2253] [Questions & Answers]
Hi guys, somebody who works with php8.1 knows what do we miss before we can update our README to state support for it?

**Comment by luigifab** [comment]
Many things. I think most of errors are: [PHP 8.1: Passing null to non-nullable internal function parameters is deprecated](https://php.watch/versions/8.1/internal-func-non-nullable-null-deprecation).

**Reply by fballiano** [reply]
I've git a rough test to OM on php8.1 developer mode this morning but it worked pretty smoothly
